tensorflow/python/tools/inspect_checkpoint.py

```diff
@@ -29,8 +29,8 @@
 FLAGS = None
 
 
-def print_tensors_in_checkpoint_file(file_name, tensor_name, all_tensors,
-                                     all_tensor_names):
+def print_tensors_in_checkpoint_file(file_name, tensor_name="", all_tensors=False,
+                                     all_tensor_names=False):
```

Comment:
Since `all_tensor_names` is added, the minimal set of changes to preserve backward compatibility is to add a default for only `all_tensor_names`. Can we do that instead?

Reply:
It would be confusing if some args have a default while others not, so I set one for all args except `file_name` which is always required.   
However, to best preserve backward compatibility, adding a default for only `all_tensor_names` is a better choice. I will do it.